tests/js/jest.config.js

```diff
@@ -16,6 +16,9 @@ module.exports = {
 		'html',
 		'text-summary',
 	],
+	globals: {
+		fetchMock: '',
```

Comment:
I'm not sure this is necessary in the config here, as Jest would be setting the global `fetchMock = ''`. See below.